cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-filter.php

```diff
@@ -401,14 +410,16 @@ public function filter_attachment_for_js( $attachment ) {
 	 * @return \WP_REST_Response
 	 */
 	public function filter_attachment_for_rest( $attachment ) {
-		if ( ! empty( $attachment->data['source_url'] ) ) {
-			$cloudinary_id = $this->media->cloudinary_id( $attachment->data['id'] );
-			if ( false !== $cloudinary_id ) {
-				$attachment->data['source_url']      = $this->media->cloudinary_url( $attachment->data['id'], false );
-				$attachment->data['transformations'] = ! ( empty( $this->media->get_transformation_from_meta( $attachment->data['id'] ) ) );
-			}
-		}
+		$cloudinary_id = $this->media->cloudinary_id( $attachment->data['id'] );
```

Comment:
Will `$attachment->data['id']` always be set?

Reply:
I think so, but I can't tell 100%. I noticed that `source_url` was missing though.

Reply:
Added a check to account for that.

Reply:
Thanks, this looks good.

Reply:
Thanks, that looks good.